runtime/parser/expression.go

```diff
@@ -816,20 +816,20 @@ func defineIdentifierExpression() {
 
 			case KeywordView:
 				// if `view` is followed by `fun`, then it denotes a view function expression
-
 				current := p.current
-				if current.Is(lexer.TokenSpace) {
-					cursor := p.tokens.Cursor()
-					p.next()
-					if p.isToken(p.current, lexer.TokenIdentifier, KeywordFun) {
-						p.nextSemanticToken()
-						return parseFunctionExpression(p, token, ast.FunctionPurityView)
-					} else {
-						p.tokens.Revert(cursor)
-						p.current = current
-					}
+				cursor := p.tokens.Cursor()
+
+				p.skipSpaceAndComments()
+
+				if p.isToken(p.current, lexer.TokenIdentifier, KeywordFun) {
+					// skip the `fun` keyword
+					p.nextSemanticToken()
+					return parseFunctionExpression(p, token, ast.FunctionPurityView)
```

Comment:
noticed a bug when resolving the merge conflict - by only checking for a space after the `view` keyword, then expressions like

```cadence
let x = view /* UwU */ fun() {}
```

fails to parse.

Reply:
Good catch! Could you please open this fix as a separate PR?